### ISSUE TITLE
Platform Engineering Maturity Model - Interfaces - wording

### ISSUE THREAD

**Issue by jkleinlercher** [cncf/tag-app-delivery#536]
In https://github.com/cncf/tag-app-delivery/blob/1ed4f4793252c39dc8d29e7d97eb13d997ec9e3f/platforms-maturity-model/v1/index.md?plain=1#L266 you write "how users interact with these platform offerings to provision, manage, and observe capabilities."

With "capabilities" I think of platform capabilities. Isn't it more about "provision, manage, and observe customer applications" on the platform?

**Comment by abangser** [comment]
Another great question! IMHO updating this to say `customer application` in place of `capabilities` would limit the platform remit to a narrow use case.

At the same time, if I understand your proposal right, what you may be saying is that this sentence would benefit from making clearer some examples of what the capabilities being observed can be?

**Comment by jkleinlercher** [comment]
yes exactly. I was a little bit confused at the first moment because I thought users of a platform don't need to observe platform capabilities. I thought the platform team should observe the platform capabilities, but maybe it is just a lack of understanding what capabilities are or how they are provided to the users of the platform.

**Comment by joshgav** [comment]
I think the original intent is indeed about interaction with platform capabilities. I think we're getting at that even platform _users_ need to observe those capabilities too, such as metrics and traces related to a database call, or information about CI/CD tasks and pipeline duration.

Let's think about how to make this clearer!

**Comment by abangser** [comment]
> maybe it is just a lack of understanding what capabilities are or how they are provided to the users of the platform

I think this is a really insightful statement and one that may be worth digging into with a blog post. This actually aligns a lot with your other open discussion (#538) and these point to a really good opportunity to write something about what it means to make a platform observable both from the perspective of the platform engineers building and operating the platform as well as for the users building on top of the platform.

Is that something you would be interested in @jkleinlercher ? I would be very happy to help support that though I am not sure I can take on a lead author role at this stage due to other commitments.

**Comment by abangser** [comment]
This repository will be locked shortly as we prepare for the [TAG Reboot](https://github.com/cncf/toc/issues/1527).

With that in mind, I wanted to just alert people who are still interested in topics like these to in the short term look for the next steps for WG Platforms in the [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) (currently #wg-platforms) and via the [Platform Engineering LinkedIn group](https://www.linkedin.com/company/platform-engineering-working-group) which is pointing at our intention to move towards a [community group](https://community.cncf.io/).

Thank you again for your contributions and we all look forward to collaborating within the new TAG and community group structures!
